cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java

```diff
@@ -908,8 +929,9 @@ public void getAppInfo(HttpRequest request, HttpResponder responder,
   @Path("/apps/{app-id}/flows")
   public void getFlowsByApp(HttpRequest request, HttpResponder responder,
                             @PathParam("app-id") String appId) {
-    programLifecycleHttpHandler.getProgramsByApp(rewriteRequest(request), responder, Constants.DEFAULT_NAMESPACE, appId,
-                                                 ProgramType.FLOW.getCategoryName());
+    programLifecycleHttpHandler.getProgramsByApp(responder,
```

Comment:
There's no v3 handler for it to delegate to, so it just calls a non-handler (helper) method in that class.